chef/cookbooks/nova/recipes/compute.rb

```diff
@@ -106,6 +106,10 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
 end
 
 case node[:nova][:libvirt_type]
+  when "zvm"
+    package "openstack-nova-zvm"
```

Comment:
no idea yet, will need to check. if you would ocmment out this #package line as well we could merge this patch already.